lib/src/widget/asset_picker_app_bar.dart

```diff
@@ -82,6 +83,8 @@ class AssetPickerAppBar extends StatelessWidget implements PreferredSizeWidget {
   /// 设置状态栏亮度层
   final Brightness? brightness;
 
+  final SystemUiOverlayStyle? systemOverlayStyle;
```

Comment:
This seems to be redundant since it can be read from `AppBarThemeData.systemOverlayStyle`.